docs/content/latest/api/ysql/datatypes/type_datetime/date-time-data-types-semantics/type-interval/interval-arithmetic/moment-moment-overloads-of-minus.md

```diff
@@ -52,24 +52,31 @@ This is the result:
 
 ```output
-2021-03-14 19:00:00-07
+IF c ◄— a - b  THEN  a ◄— c + b
 ```
 
-You don't get back the _timestamptz_ value that you started with.
+The critical feature of this example  with respect to what seems to be a wrong result is that 18:00 on 13-Mar-2021 in the 'America/Los_Angeles' timezone is _before_ the "spring forward" moment when Daylight Savings Time starts, and that 20:00 and 18:00 on 14-Mar-2021 in that timezone is _after_ the "spring forward" moment.
+
+It's possible to understand, and predict, this outcome (and other hybrid _interval_ arithmetic outcomes like it) by understanding the three different kinds of semantics for the moment-_interval_ overloads of the `+` and `-` operators for pure _mm_ _interval_ values, pure _dd_ _interval_ values, and pure _ss_ _interval_ values.
+
+These rules are explained in the section [The moment-_interval_ overloads of the "+" and "-" operators for _timestamptz_, _timestamp_, and _time_](../moment-interval-overloads-of-plus-and-minus/). The rules for the trickiest case, moment-_interval_ arithmetic using a _pure days interval_ value, are explained in exhaustive detail in the section [Sensitivity of timestamptz-interval arithmetic to the current timezone](../../../../timezones/timezone-sensitive-operations/timestamptz-interval-day-arithmetic/).
 
-The critical feature of this example that least to what seems to be a wrong result is that 18:00 on 13-Mar-2021 in the 'America/Los_Angeles' timezone is before the "spring forward" moment when Daylight Savings Time starts, and that 20:00 and 19:00 on 14-Mar-2021 in that timezone are both after the "spring forward" moment.
+However, when you have a hybrid _dd_ and _mm_ _interval_ value, you need also to understand the priority rule: in which order are the different variations of addition/subtraction semantics done?
 
-It's possible to understand, and predict, this outcome (and other hybrid _interval_ arithmetic outcomes like it) by understanding the three different kinds of semantics for the moment-_interval_ overloads of the `+` and `-` operators for pure _mm_ _interval_ values, pure _dd_ _interval_ values, and pure _ss_ _interval_ values. But when you have a hybrid _dd_ and _mm_ _interval_ value, you need also to understand the priority rule: in which order are the different variations of addition/subtraction semantics done?
+{{< tip title="Avoid arithmetic that uses hybrid 'interval' semantics." >}}
+Yugabyte staff members have discussed this carefully and believe that the attempt to understand and memorize these rules is counterproductive—and therefore foolish. Rather, you should decide which of the three kinds of semantics your application needs and arrange that you produce only pure, rather than hybrid, _interval_ values. It recommends, therefore, that  you adopt the practice that the section [Custom domain types for specializing the native _interval_ functionality](../../custom-interval-domains/) explains.
```

Comment:
```suggestion
Yugabyte staff members have discussed this carefully and believe that the attempt to understand and memorize these rules is counterproductive—and therefore foolish. Rather, you should decide which of the three kinds of semantics your application needs and arrange that you produce only pure, rather than hybrid, _interval_ values. Yugabyte recommends, therefore, that  you adopt the practice that the section [Custom domain types for specializing the native _interval_ functionality](../../custom-interval-domains/) explains.
```

Reply:
("We recommend" is also OK here)